app/controllers/actions/AuthenticatedActionRefiner.scala

```diff
@@ -50,6 +52,8 @@ class AuthenticatedActionRefiner @Inject()(
       case _ =>
         throw new Exception("Nino not found")
     } recover {
+      case _: BearerTokenExpired | _: InvalidBearerToken =>
+        Left(Redirect("/sign-in"))
```

Comment:
I am not sure about this. What is the root here? Is really going to bas gateway and not to marriage-allowance-application?

Reply:
will take a look to ensure this works as I expect..

Reply:
yeah good spot as a minimum needs to be /gg/sign-in for a start :)